src/dso_api/dynamic_api/routers.py

```diff
@@ -68,8 +69,9 @@ def _initialize_viewsets(self) -> List[Type[DynamicModel]]:
         # This makes sure the 'to' field is resolved to an actual model class.
         for app_label, models_by_name in self.all_models.items():
             for model in models_by_name.values():
-                dataset_name = model.get_dataset_id()
-                url_prefix = f"{dataset_name}/{model.get_table_id()}"
+                dataset_name = slugify(model.get_dataset_id(), sign="_")
```

Comment:
Misschien het is beter om `get_dataset_id` en `get_table_id` aan te passen op daar `slugify` te roepen? Anders krijgen we dezelfde probleem in andere plekken...

Reply:
Ja. Ik twijfelde of er wellicht nog ergens code afhankelijk was van de oorsprokelijke naam (met evt. een spatie erin). Maar, dat lijkt er niet op. Dan beter om het bij de bron aan te passen, ga ik doen.

Reply:
Die code zit in de schema-tools. Ik verwijderd dit pull request en maak een nieuwe aan voor de schema-tools.